client/src/Pages/Profile/SideBar/index.js

```diff
@@ -61,7 +67,7 @@ function SideBar() {
         </Tabs>
       </div>
       <TabPanel value={value} index={0} className={classes.mainContent}>
-        user info
+        <UserInfo getUserName={getUserName} />
```

Comment:
Here for example, you could pass setUsername directly. Also I think the prop should be called setUsername